pkgs/top-level/python-packages.nix

```diff
@@ -9794,20 +9794,19 @@ in {
   jabberbot = callPackage ../development/python-modules/jabberbot {};
 
   jedi = callPackage ../development/python-modules/jedi { };
-
   jellyfish = buildPythonPackage rec {
```

Comment:
jellyfish upgrade needs to be dropped from this PR because it is currently in the staging branch

Reply:
@FRidh it seems that `us` requires `jellyfish-0.5.6`. Please advise.

Reply:
I've just merged staging into master, so master has 0.5.6 in it. You can rebase on master.

Reply:
Rebased just now